src/persistence/Elsa.Persistence.EntityFrameworkCore/DbContexts/DesignTimeDbContextFactoryBase.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Design;
+
+namespace Elsa.Persistence.EntityFrameworkCore.DbContexts
+{
+    public abstract class DesignTimeDbContextFactoryBase<TDbContext> : IDesignTimeDbContextFactory<TDbContext> where TDbContext : DbContext
+    {
+        public TDbContext CreateDbContext(string[] args)
+        {
+            var optionsBuilder = new DbContextOptionsBuilder<TDbContext>();
+            var migrationAssembly = typeof(TDbContext).Assembly.FullName;
+            var connectionString = Environment.GetEnvironmentVariable("EF_CONNECTIONSTRING");
+
+            if(connectionString == null)
+            {
+                var providerName = typeof(TDbContext).Name.Replace("Context", "");
+                throw new InvalidOperationException($"Set the EF_CONNECTIONSTRING environment variable to a valid {providerName} connection string.");
+            }
+
+            optionsBuilder.UseSqlite(
+                connectionString,
+                x => x.MigrationsAssembly(migrationAssembly)
```

Comment:
Because this only uses UseSqlite then no other migrations can be created for the other providers.
We could make this work by allowing the implementations to configure the options as an abstract method that must be implemented, that can then set the provider.

Reply:
Ignore!
I can see you reverted back with the correct options builder applied in each factory in another PR :)

Reply:
Exactly. I realized my mistake right after I merged it 😅

Reply:
Thanks for looking though!